### PR TITLE
block more spam scrapers

### PR DESCRIPTION
**Pull request by pushcx**
Thanks for providing a sample of how to request spam tools not plagiarize one's site. It's been [useful for Lobste.rs](https://github.com/lobsters/lobsters/blob/d778ead68807323be4e8cf9756334a1d21826c05/public/robots.txt) and [the discussion](https://lobste.rs/s/ybowdq/great_gpt_firewall) turned up some more abusers.

I added these others to the sample and scraper. I didn't re-run the crawl because it sounds like you want to create a git history with monthly commits, but I can if you'd like it added to this PR.

I noticed `ccbot` was in the sample `robots.txt` but not present in `scrape.py`, so I added it.

I noticed `anthropic-ai` and `google-extended` swapped positions in the two files so I matched them. I think you should consider alphabetically sorting these for future convenience, but it seemed a little too far in the direction of a personal style choice to include in this PR.

### CODE REVIEW

**Comment by samber** [comment]
oh nice, thank you!

I will run a new scrape next weekend and check the result.

**Comment by macie** [comment]
@pushcx What about [FacebookBot](https://developers.facebook.com/docs/sharing/bot)? It's on the Neil Clarke list

**Comment by pushcx** [comment]
@macie Whoops, it's an oversight. I've added it.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ User-agent: anthropic-ai
 # Common Crawl
 User-agent: CCBot
 
+# Omglibot: webz.io
+User-agent: Omgilibot
```
Comment (samber) [inline README.md:31]: A lot of robots.txt restricts those user agents, but I don't think this is specific to LLM crawlers.

I would argue against this one, and consider GPTBot as a better marker for this list.

At least, we should add a comment to explain this is a general-purpose crawler.

WDYT?

File: README.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ User-agent: anthropic-ai
...
+User-agent: Omgili
+
+# Facebook: LLaMA2
+User-agent: FacebookBot
+
+# ByteDance: Duobao
+User-agent: Bytespider
```
Comment (samber) [inline README.md:38]: Same here.

**Comment by samber** [comment]
Can you add AmazonBot please? It has been designed for Alexa, so I suppose it should be in this list.

https://developer.amazon.com/support/amazonbot

File: README.md (hunk 1 above)
Comment (macie) [reply README.md:31]: Webz.io advertises itself as a source of high quality data for model training: https://webz.io/datasets/premium-datasets

But you may be right. The list of crawlers used to train AI models is open-ended. If I had a big dataset, I'd try to monetize it now, during the hype stage :smile:

**Comment by macie** [comment]
I found a list with more AI bots: https://darkvisitors.com/
Not all positions are linked to the source, and the list contains statistics without explanation.

**Comment by pushcx** [comment]
That Dark Visitors site makes it clear how quickly these spammers are growing. I hope this PR is a useful addition to your list, but I'm not going to try to keep up with them. I'd rather contribute to a site like Dark Visitors that better aligns with my understanding of this abuse. Best wishes.

**Comment by samber** [comment]
@pushcx Sure, no problem.

I did not know this site before and am not sure we have the same goal. (mesuring blocking per user-agent vs website category).

I will merge your PR with some updates. ✅

**Pull request merged**